tests/meta/clean/test_remove_old_dirs.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+from pathlib import Path
+from typing import Dict, List
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.meta.clean.flow_cell_run_directories import RunDirFlowCell
+from cg.store import Store
+from housekeeper.store.models import File, Tag, Version
+from tests.store_helpers import StoreHelpers
+
+
+def test_archive_flow_cell_run_directory(
+    caplog,
+    real_housekeeper_api: HousekeeperAPI,
+    dragen_flow_cell_dir: Path,
+    timestamp: datetime.datetime,
+    base_store: Store,
+    helpers: StoreHelpers,
+):
+    # GIVEN an old flow cell which is not tagged as archived but whose sample sheet is included in Housekeeper
+
+    bundle_data: Dict = {"name": "HLG5GDRXY", "created": timestamp, "files": []}
+
+    helpers.ensure_hk_bundle(store=real_housekeeper_api, bundle_data=bundle_data)
+
+    real_housekeeper_api.add_and_include_file_to_latest_version(
+        bundle_name="HLG5GDRXY",
+        file=Path(dragen_flow_cell_dir, "SampleSheet.csv"),
+        tags=["samplesheet", "HLG5GDRXY"],
+    )
```

Comment:
What would be the benefit of having it as a fixture? Won't it be maintained for the whole test suite then since it is not used anywhere else that I know of?